onadata/apps/logger/models/xform.py

```diff
@@ -561,7 +561,7 @@ def get_additional_geopoint_xpaths(cls, xpath, remove_group_name=False):
         prefix = ""
         name = xpath
         if match:
-            prefix = "" if remove_group_name else match.groupdict()["prefix"]
```

Comment:
Removing this line since we are generating XPaths; If you remove the group name for an XPath it's no longer an accurate XPath to the actual data.